Missing Application plugins & Graphics' lineStyle types

I just updated to PixiJS v6 RC and wanted to check if the updated types in v6 broke our Angular project and I get this errors that certain attributes don't exist anymore on certain types. If I check the v6 docs, the attributes are still there and not deprecated. Then I saw, that the pixi.js.d.ts is missing from node_modules/pixijs and maybe that confuses the typescript compiler.

I get this error from VS Code:

`"Unable to open 'pixi.js.d.ts': Unable to read file '/Users/toms/Coding/WaveModeler/node_modules/pixi.js/pixi.js.d.ts' (Error: Unable to resolve non-existing file '/Users/toms/Coding/WaveModeler/node_modules/pixi.js/pixi.js.d.ts')."`

And a lot of compilation errors like this:
`ERROR in src/app/main/processTabContent/editor/editor.component.ts:102:18 - error TS2339: Property 'loader' does not exist on type 'Application'.
this.pixiApp.loader.load(() => this.pixiApp.renderer.render(this.pixiApp.stage));`

### Environment

- **`pixi.js` version**: 6.0.0-rc
- **OS & Version**: macOS 11.1 Big Sur
- **Angular Version**: Angular 10
- **Typescript Version**: 3.9.7

@bigtimebuddy I'll try to make an StackBlitz example :) 

In the meantime: Those aren't the only errors. Here are a few more. I guess all connected to the error with the wrong *.d.ts

`src/app/main/toolbar/export/StageExporter.ts:46:42 - error TS2339: Property 'extract' does not exist on type 'Renderer | AbstractRenderer'. Property 'extract' does not exist on type 'AbstractRenderer'.
const dataURL = pixiApp.renderer.extract.canvas(stage).toDataURL();`

`src/app/main/toolbar/export/StageExporter.ts:81:25 - error TS2339: Property 'ticker' does not exist on type 'Application'.
pixiApp.ticker.remove(exportFunc);`

`src/shapes/flowObjectShapes/activity/ActivityShape.ts:69:47 - error TS2554: Expected 0-1 arguments, but got 4.
this.lineStyle(ActivityStyle.borderWidth, ActivityStyle.borderColor, 1, 0.5);`

--- a/packages/app/global.d.ts
+++ b/packages/app/global.d.ts
@@ -1,0 +1,15 @@
+declare namespace GlobalMixins
+{
+    // eslint-disable-next-line @typescript-eslint/no-empty-interface
+    interface Application
+    {
+        resizeTo: Window|HTMLElement;
+        resize(): void;
+    }
+
+    // eslint-disable-next-line @typescript-eslint/no-empty-interface
+    interface IApplicationOptions
+    {
+        resizeTo: Window|HTMLElement;
+    }
+}

--- a/packages/app/src/Application.ts
+++ b/packages/app/src/Application.ts
@@ -14,7 +14,7 @@ export interface IApplicationPlugin {
 export interface IApplicationOptions extends IRendererOptionsAuto, GlobalMixins.IApplicationOptions {}
 
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface Application extends GlobalMixins.DisplayObject {}
+export interface Application extends GlobalMixins.Application {}
 
 /**
  * Convenience class to create a new PIXI application.

--- a/packages/app/src/Application.ts
+++ b/packages/app/src/Application.ts
@@ -10,12 +10,11 @@ export interface IApplicationPlugin {
     destroy: (...params: any[]) => any;
 }
 
-export interface IApplicationOptions extends IRendererOptionsAuto {
-    autoStart?: boolean;
-    sharedTicker?: boolean;
-    sharedLoader?: boolean;
-    resizeTo?: Window | HTMLElement;
-}
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface IApplicationOptions extends IRendererOptionsAuto, GlobalMixins.IApplicationOptions {}
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface Application extends GlobalMixins.DisplayObject {}
 
 /**
  * Convenience class to create a new PIXI application.

--- a/packages/loaders/global.d.ts
+++ b/packages/loaders/global.d.ts
@@ -10,4 +10,14 @@ declare namespace GlobalMixins
     {
 
     }
+
+    interface Application
+    {
+        loader: import('@pixi/loaders').Loader;
+    }
+
+    interface IApplicationOptions
+    {
+        sharedLoader: boolean;
+    }
 }

--- a/packages/ticker/global.d.ts
+++ b/packages/ticker/global.d.ts
@@ -1,0 +1,15 @@
+declare namespace GlobalMixins
+{
+    interface Application
+    {
+        ticker: import('@pixi/ticker').Ticker;
+        stop(): void;
+        start(): void;
+    }
+
+    interface IApplicationOptions
+    {
+        autoStart: boolean;
+        sharedTicker: boolean;
+    }
+}

